components/ModEditFacilityOrProfessionalTopbar.vue

```diff
@@ -64,21 +74,33 @@
                 <div
                     class="flex flex-col aspect-square h-96 items-center justify-around bg-primary-inverted p-10 rounded"
                 >
-                    <span
-                        v-show="moderationScreenStore.editFacilityScreenIsActive()"
-                        class="font-bold text-3xl"
-                    >
-                        {{ $t('modEditFacilityOrHPTopbar.deleteConfirmationFacility',
-                              { id: selectedId, facility: facilitiesStore.selectedFacilityData?.nameEn }) }}
-                    </span>
-                    <button
-                        class="bg-primary p-4 rounded-full my-8 font-semibold text-xl"
-                        type="button"
-                        @click="deleteFacilityOrHealthcareProfessional"
-                    >
-                        {{
-                            $t('modEditFacilityOrHPTopbar.deleteButtonText') }}
-                    </button>
+                    <template v-if="modalType === 'unsavedChanges'">
+                        <span class="font-bold text-3xl">{{ $t('modEditFacilityOrHPTopbar.hasUnsavedChanges') }}</span>
```

Comment:
We can use a div here since it's a block element. 
Basically, when you have inline text you want `span`, otherwise `divs` are usually the best